.github/check-wiki-changelog.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+source .github/bot-pr-base.sh
+
+echo "Checking whether the PR contains the 1:ST:skip-changelog-wiki-check label"
+SKIP_CHECK=$(api_get "$PR_URL" | jq -r 'any( [.labels | .[] | .name ] | .[] ; . == "1:ST:skip-changelog-wiki-check")')
+
+if [[ "$SKIP_CHECK" == "true" ]]; then
+  echo "The PR contains the 1:ST:skip-changelog-wiki-check label. Skipping the wiki check."
+elif [[ "$HEAD_BRANCH" =~ ^release.* ]]; then
```

Comment:
nit: I think the naming scheme was always release-1.2.3, so we can be a bit more precise
```suggestion
elif [[ "$HEAD_BRANCH" =~ ^release-.* ]]; then
```